### PR TITLE
[21.05] firefox: introduce nspr_latest, update nss_latest

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

```
ERROR: Requested 'nspr >= 4.32' but version of NSPR is 4.30.0 
ERROR: Requested 'nss >= 3.68' but version of NSS is 3.67.0
```

Fixup #133305 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/nspr/latest.nix [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{ lib, stdenv, fetchurl
+, CoreServices ? null
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/nspr/latest.nix:2]:
```suggestion
, CoreServices
```

if this is null something is wrong.

Comment (mweinelt) [reply pkgs/development/libraries/nspr/latest.nix:2]: Verbatim copy of the package from master. Also part of the stable package. I'd rather not change it in the port.

**Comment by mweinelt** [comment]
I have builds for all three firefox flavors running. They all passed the configure phase, so we could merge this early, to fix the situation quickly. Or we wait until I have completed all tests. I don't mind either way.

**Comment by mweinelt** [comment]
The 91 and 91esr tests have completed. The 78esr test is looping in `wait_for_sound`.

**Pull request merged**